src/components/ProfilePage/QuestsOverview/QuestsOverview.tsx

```diff
@@ -0,0 +1,94 @@
+import { useTranslation } from 'react-i18next';
+import { useOngoingQuests } from 'src/hooks/useOngoingQuests';
+import type { Quest } from 'src/types/loyaltyPass';
+import { checkInclusion } from '../checkInclusion';
+import { QuestCard } from '../QuestCard/QuestCard';
+import {
+  QuestsOverviewContainer,
+  QuestsOverviewTitle,
+} from './QuestsOverview.style';
+
+interface QuestCarouselProps {
+  pastCampaigns?: string[];
+  traits?: string[];
+  // traits?: Trait[];
+}
+
+export const QuestsOverview = ({
+  pastCampaigns,
+  traits,
+}: QuestCarouselProps) => {
+  const { t } = useTranslation();
+  const { quests, isLoading: isQuestsLoading, url } = useOngoingQuests();
+
+  return (
+    <QuestsOverviewContainer>
+      <QuestsOverviewTitle variant="headerMedium">
+        {t('missions.available')}
+      </QuestsOverviewTitle>
+      {/** render quests */}
+      {!isQuestsLoading
+        ? quests?.map((quest: Quest, index: number) => {
+            const baseURL = quest?.attributes.Image?.data?.attributes?.url;
+            const imgURL = baseURL && new URL(baseURL, url.origin);
+            const rewards = quest?.attributes.CustomInformation?.['rewards'];
```

Comment:
I remember adding a comment about this somewhere, attributes might be null, either it needs to be asserted above and you can remove all the attributes? or you need to add the `?`

Reply:
thanks